src/cli.rs

```diff
@@ -123,6 +123,10 @@ pub fn handle_arguments() -> anyhow::Result<i32> {
         }
     });
 
+    if opts.measure_stack {
+        log::warn!("use of deprecated option `--measure-stack`: Has no effect anymore and will vanish on next minor release")
```

Comment:
I think we have to wait for a major release to take away command line arguments?

Reply:
The current version is `v0.3.9`. In my understanding of semver the next breaking release will be `v0.4.0`.

If the version would be e.g. `v1.3.9`, the next breaking release would be `2.0.0`.

Correct me if I am wrong.

Reply:
That is correct.

Reply:
Which means 0.4 is a "major" release, even though you updated the middle digit.

Reply:
Although semver.org says:

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

So technically you can do what you like.

Reply:
Interesting. My brain parses it as major.minor.patch, so I though it would be a minor release if we bump the middle number. I will change it to "breaking release".